Thumbprint/Xcconfig/Thumbprint-Debug.xcconfig

```diff
@@ -3,7 +3,6 @@
 ALWAYS_SEARCH_USER_PATHS = NO
 APPLICATION_EXTENSION_API_ONLY = YES
 ARCHS[sdk=iphoneos*] = arm64
-ARCHS[sdk=iphonesimulator*] = x86_64
```

Comment:
I suspected the presence of this line might've been an issue.

Reply:
Just want to confirm this won't impact not M1s?

Reply:
i tested using rosetta, but i can verify on an intel mac to be sure, but pretty sure that's how we have the main iOS project set up as well.

Reply:
Manually tested on intel, and also CI is intel based i believe.